packages/playwright-core/src/server/registry/index.ts

```diff
@@ -913,13 +913,13 @@ export class Registry {
 
   async install(executablesToInstall: Executable[], forceReinstall: boolean) {
     const executables = this._dedupe(executablesToInstall);
-    await fs.promises.mkdir(registryDirectory, { recursive: true });
-    const lockfilePath = path.join(registryDirectory, '__dirlock');
-    const linksDir = path.join(registryDirectory, '.links');
+    await fs.promises.mkdir(registryDirectory(), { recursive: true });
```

Comment:
let's use a local variable for `registryDirectory()` here and in other methods where it's called more than once